Test.c

```diff
@@ -8,7 +8,78 @@
 // ------------------------------------------------------------------------------------------------
 
 #include <stdio.h>
+#include <stdbool.h>
+#include "Header.h"
 
-int main () { 
-	printf ("Hello World!\n");
+typedef struct {
+   ComplexNumber num1;
+   ComplexNumber num2;
+   ComplexNumber out;
+}Test;
+
+bool IsEqual (ComplexNumber num1, ComplexNumber num2) {
+   return ((num1.Real == num2.Real) && (num1.Img == num2.Img));
+}
+
+PrintNum (ComplexNumber num, char* str) {
+   printf ("  %5.3f + i %5.3f%s", num.Real, num.Img, str);
+}
+
+void Test_Op_Mul () {

```

Comment:
The test is too verbose. You were asked to just mention what is being tested and the result.